cogs/help.py

```diff
@@ -0,0 +1,167 @@
+                f"The prefix for this channel is `{prefix}`\n{cog.description}"
+            ),
+            colour=ctx.bot.colors["blue"],
+        )
+        embed.set_author(name=str(ctx.message.author), icon_url=str(ctx.message.author.avatar_url))
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        for command in await self.filter_commands(cog.get_commands()):
+            embed.add_field(
+                name=f"{prefix}{self.get_command_signature(command)}",
+                value=command.help,
+                inline=False,
+            )
+        embed.set_footer(
+            text=f"Are you interested in {cog.qualified_name} ?",
+            icon_url=str(ctx.bot.user.avatar_url),
+        )
+        await ctx.send(embed=embed)
+
+    async def send_command_help(self, command: commands.Command) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=f"{prefix}{self.get_command_signature(command)}",
+            description=f"Command syntax : `<Those arguments are required>`. `[Those aren't]`\n{command.help}",
+            colour=ctx.bot.colors["blue"],
+        )
+        if command.aliases:
+            embed.add_field(name="Aliases :", value="\n".join(command.aliases))
+        embed.set_author(
+            name=str(ctx.message.author),
+            icon_url=str(ctx.message.author.avatar_url),
+        )
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        if command.hidden:
+            embed.set_footer(
+                text=f"Wow, you found {command.name} !",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        else:
+            embed.set_footer(
+                text=f"Are you interested in {command.name} ?",
+                icon_url=str(ctx.bot.user.avatar_url),
+            )
+        await ctx.send(embed=embed)
+
+    async def send_group_help(self, group: commands.Group) -> None:
+        ctx = self.context
+        prefix = ctx.prefix
+        embed = discord.Embed(
+            title=f"Help for group {prefix}{self.get_command_signature(group)}",
+            description=(
+                "Command syntax : `<Those arguments are required>`. `[Those aren't]`\n"
+                f"{group.help}"
+            ),
+            colour=ctx.bot.colors["blue"],
+        )
+        for command in await self.filter_commands(group.commands, sort=True):
+            embed.add_field(
+                name=f"{prefix}{self.get_command_signature(command)}",
+                value=command.help,
+                inline=False,
+            )
+        embed.set_author(
+            name=str(ctx.message.author),
+            icon_url=str(ctx.message.author.avatar_url),
+        )
+        embed.set_thumbnail(url=str(ctx.bot.user.avatar_url))
+        if group.hidden:
+            embed.set_footer(
+                text=f"Wow, you found {group.name} !",
```

Comment:
It's quite unusual to put space before exclamation marks.

Note: This also applies to every such occurrence in this file.
```suggestion
                text=f"Wow, you found {group.name}!",
```